app/assets/javascripts/app/pages/admin_pods.js

```diff
@@ -4,10 +4,48 @@ app.pages.AdminPods = app.views.Base.extend({
   templateName: "pod_table",
 
   tooltipSelector: "th i",
+  events: {
+    "click #show_all_pods": "showAllPods",
+    "click #show_active_pods": "showActivePods",
+    "click #show_invalid_pods": "showInvalidPods"
+  },
 
   initialize: function() {
     this.pods = new app.collections.Pods(app.parsePreload("pods"));
     this.rows = []; // contains the table row views
+    this.podfilter = "active";
+  },
+
+  showAllPods: function() {
+    this.podfilter = "";
+    this.postRenderTemplate();
+    this.$("#show_all_pods").addClass("active");
+    this.$("#show_active_pods").removeClass("active");
+    this.$("#show_invalid_pods").removeClass("active");
```

Comment:
I don't know if there is a better solution than manually adding and removing the `active` class everywhere? Because that doesn't scale that well if there are are more buttons (as it looks like you plan to add one for blocked pods in the future).

I don't know if there is a possibility to group these buttons somehow so only one can be active at once, no matter how many buttons exist (like radio buttons in forms?). Maybe just remove it from all buttons and just add it back to the one that was clicked?